### PR TITLE
adu: used /etc/sw-versions file for swupdate_consent_handler

### PR DESCRIPTION
**Pull request by JoergZeidler**
- omnect-device-service: updated to 0.14.12

### CODE REVIEW

File: recipes-azure-iot/iot-hub-device-update/iot-hub-device-update_1.0.2.bb [hunk 1]
```diff
@@ -1,167 +1,164 @@
...
-  ${sysconfdir}/omnect/consent/swupdate/user_consent.json \
-  ${sysconfdir}/omnect/consent/swupdate/installed_criteria \
-  "
-
-GROUPADD_PARAM:${PN} += " \
-  -r adu; \
-  -r do; \
-"
-USERADD_PARAM:${PN} += "--no-create-home -r -s /bin/false -G aziotcs,aziotid,aziotks,do -g adu adu;"
+FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:${LAYERDIR_omnect}/files:"
```
Comment (JanZachmann) [inline recipes-azure-iot/iot-hub-device-update/iot-hub-device-update_1.0.2.bb:1]: why is the whole file a diff?

Comment (JoergZeidler) [reply recipes-azure-iot/iot-hub-device-update/iot-hub-device-update_1.0.2.bb:1]: autoremove of line endings

**Review by JanZachmann — APPROVED** [review]

File: recipes-azure-iot/iot-hub-device-update/iot-hub-device-update_1.0.2.bb (hunk 1 above)
Comment (mlilien) [reply recipes-azure-iot/iot-hub-device-update/iot-hub-device-update_1.0.2.bb:1]: please revert that

**Review by mlilien — COMMENTED** [review]
https://github.com/omnect/meta-omnect/pull/378#discussion_r1443029074

File: recipes-azure-iot/iot-hub-device-update/iot-hub-device-update_1.0.2.bb (hunk 1 above)
Comment (JoergZeidler) [reply recipes-azure-iot/iot-hub-device-update/iot-hub-device-update_1.0.2.bb]: turned back to original line ending cr+lf

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [kirkstone-0.28.7] Q4 2023
```
Comment (mlilien) [inline CHANGELOG.md:8]: Q1 2024

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
...
+- iot-hub-device-update: used /etc/sw-versions file for swupdate_consent_handler
```
Comment (mlilien) [inline CHANGELOG.md:9]: used -> use

File: CHANGELOG.md (hunk 2 above)
Comment (JoergZeidler) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 3 above)
Comment (JoergZeidler) [reply CHANGELOG.md]: done

**Review by mlilien — APPROVED** [review]

**Pull request merged**